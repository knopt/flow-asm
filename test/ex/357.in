1
27
4.044146857272267
0.591054901150672 
8.790386996498839 
-1.4163684014194473 
-8.441810759785414 
7.004422987493481 
-9.144808922043035 
8.196453107494182 
7.1130664954796 
8.702731407270406 
-1.7177884870263256 
-6.464283367801578 
0.38875601671795046 
-6.9142735037044805 
2.410504253228737 
-7.311285302686919 
-8.973395877958625 
5.388104933126778 
8.87496598767904 
-2.3721921185781465 
-9.63300463293014 
4.737280042548473 
5.38128394368729 
-6.914509300277354 
-1.159668071245811 
8.043954169569759 
-6.071956989228071 
-4.812025197792373 

6.021988963202315 -9.966956091479478 2.705206964889088 -1.880998903666919 -6.204223117150917 -2.5348849209613444 0.9257450180951743 5.60008791782707 -0.6670757824353011 0.899432963309156 4.354021460444015 -3.904347044053571 -3.9639600194554747 8.426669874046887 -8.73600898292835 -6.435124207744463 9.772681952671729 5.608618065454724 -2.6710641960169434 -0.3069510906828441 2.372024166971725 5.613989872485806 -5.218793041144338 -1.2755605070655243 3.684855944238109 2.8285416295483596 -8.833566661204008 
