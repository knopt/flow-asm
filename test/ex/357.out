13.01570597049238 
-177.30915390090365 
-8.341281525891347 
163.09914392160698 
-248.7022979006942 
205.64613017533566 
-149.61853214669847 
-44.68190279518782 
-121.86632682659503 
60.6177574418351 
124.32945884598365 
-75.7893119034921 
146.47613462677876 
-121.14375992731118 
86.71137443003349 
142.8655043607488 
-67.78174936993683 
-106.98450846373564 
53.16819874294142 
192.26746824135196 
-77.19332551022461 
-101.04621555757538 
146.4144605519141 
15.495291870787707 
-162.6738771539718 
120.39476275908812 
4.728495659418012 
